File: 251004/Yelkin/4-5/CommentService/src/main/java/by/yelkin/CommentService/repository/CommentRepository.java
package by.yelkin.CommentService.repository;

import by.yelkin.CommentService.entity.Comment;
import org.springframework.data.cassandra.repository.CassandraRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface CommentRepository extends CassandraRepository<Comment, String> {
}

File: 251003/Kukhatskavolets/microblog/discussion/src/main/kotlin/by/kukhatskavolets/discussion/repositories/CommentRepository.kt
package by.kukhatskavolets.discussion.repositories

import by.kukhatskavolets.discussion.entities.Comment
import by.kukhatskavolets.discussion.entities.CommentKey
import org.springframework.data.cassandra.repository.CassandraRepository
import org.springframework.stereotype.Repository

@Repository
interface CommentRepository : CassandraRepository<Comment, CommentKey>
